### PR TITLE
Remove "elastic" methods and data structures

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [ ] tests added
- [x] commits conform to the [Commit message format](https://github.com/qlik-oss/gopherciser/blob/master/.github/CONTRIBUTING.md#commit)
- [ ] documentation updated

**Squash commit message**

* Remove all elastic actions
* Remove all elastic related methods and data structures
* Add possibility to register custom state carriers to state object

**Info**

Optional informative text (not to be included in commit message) relevant to reviewers.

### CODE REVIEW

File: generatedocs/data/actions/disconnectenvironment/description.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## DisconnectEnvirnment action
```
Comment (SeBBBe) [inline generatedocs/data/actions/disconnectenvironment/description.md:1]: Environment*

File: session/sessionstate.go [hunk 2]
```diff
@@ -988,3 +992,20 @@ func (state *State) ClearSubscribedObjects(IDs []string) error {
 func (state *State) UpdateFeatureMap(host string, actionState *action.State) {
 	state.Features.UpdateFeatureMap(state.Rest, host, actionState, state.LogEntry)
 }
+
+// AddCustomState add custom data handler to state object
+func (state *State) AddCustomState(key string, value interface{}) {
```
Comment (SeBBBe) [inline session/sessionstate.go:997]: Don't really understand what these are supposed to be used for

**Comment by atluq** [comment]
Structs in `datafilesresponse.go` and `collection.go` is unused after the move, maybe they should also be moved.

File: session/sessionstate.go (hunk 2 above)
Comment (DnlLrssn) [reply session/sessionstate.go:997]: When you need to add data propagated between actions when using qlik-oss/gopherciser as a library

**Comment by DnlLrssn** [comment]
> 
> 
> Structs in `datafilesresponse.go` and `collection.go` is unused after the move, maybe they should also be moved.

ok, thanks , will take a look

File: session/sessionstate.go (hunk 2 above)
Comment (SeBBBe) [reply session/sessionstate.go:997]: Between actions??? Crazy

**Review by SeBBBe — APPROVED** [review]

**Review by atluq — APPROVED** [review]

**Pull request merged**